test/utils/path.spec.js

```diff
@@ -1,8 +1,48 @@
 import {expect} from 'chai'
-import {parse} from '../../app/scripts/utils/path'
+import Path, {parse} from '../../app/scripts/utils/path'
```

Comment:
All the gory details: http://www.2ality.com/2014/09/es6-modules-final.html

short version:

```js
import Path, {parse} from 'some'
// translates roughly into
var Path = require('some')
var parse = Path.parse
```